doc/cookbook/source/examples/regression/multiple_kernel_learning_regression.rst

```diff
@@ -0,0 +1,56 @@
+========================
```

Comment:
can you remove "regression" from filename
Ah crap, need to change data filename as well, didnt realise earlier